training_review/models.py

```diff
@@ -29,6 +29,7 @@ class Meta:
                                on_delete=models.CASCADE)
     user = models.ForeignKey(User, verbose_name='创建用户',
                              on_delete=models.CASCADE)
+    user_name = models.CharField(verbose_name='用户姓名', max_length=64)
```

Comment:
Do not add unnecessary field, it's redundant since `user.first_name` is already there. As I said before, you should use [SlugRelatedField](https://www.django-rest-framework.org/api-guide/relations/#slugrelatedfield)